src/main/java/io/vertx/openapi/impl/Utils.java

```diff
@@ -61,6 +61,22 @@ public static Future<JsonObject> readYamlOrJson(Vertx vertx, String path) {
     });
   }
 
+    /**
+   * Reads YAML string and transforms it into a JsonObject.
+   *
+   * @param path  The yamlString proper YAML formatted STring
+   * @return A succeeded Future holding the JsonObject, or a failed Future if the file could not be parsed.
+   */
+  public static Future<JsonObject> yamlStringToJson(String yamlString) {
```

Comment:
Re-use this method in line 52, then it is also tested indirectly and I don't have to insist on a test.

Reply:
adjusted as requested